nautobot/dcim/tests/test_cablepaths.py

```diff
@@ -44,12 +43,13 @@ def setUpTestData(cls):
 
         manufacturer = Manufacturer.objects.create(name="Generic", slug="generic")
         device_type = DeviceType.objects.create(manufacturer=manufacturer, model="Test Device")
-        device_role = DeviceRole.objects.create(name="Device Role", slug="device-role")
+        # TODO(timizuo): Device Role Reassign
```

Comment:
What work is needed here?

Reply:
It would be replaced with Role, in the next PR where i write test